doc/sphinx/src/models.rst

```diff
@@ -3,3 +3,409 @@
+parameter, and :math:`E` is the specific internal energy. In this sense, an EOS
+of this form uses the Gruneisen parameter to describe the pressure behavior of
+the EOS away from the reference curve. Coupled with a relationship between
+energy and temperature (sometimes as simple as a constant heat capacity), the
+complete equation of state can be constructed.
+
+To some degree it is the complexity of the reference state and the heat
+capacity that will determine an EOS's ability to capture the complex behavior of
+a material. At the simplest level, the ideal gas EOS uses a reference state at
+zero pressure and energy, while more complex equations of state such as the
+Davis EOS use the material's isentrope. In ths way, the reference curve
+indicates the conditions under which you can expect the EOS to represent the
+intended behavior.
+
+Nomenclature
+---------------------
+
+The EOS models in ``singularity-eos`` are defined for the following sets of
+dependent and independent variables through various member functions described
+in the :doc:`EOS API <using-eos>`.
+
++--------------------------+----------------------+------------------------+
+| Function                 | Dependent Variable   | Independent Variables  |
++==========================+======================+========================+
+| :math:`T(\rho, E)`       | Temperature          | Density, Energy        |
++--------------------------+----------------------+                        |
+| :math:`P(\rho, E)`       | Pressure             |                        |
++--------------------------+----------------------+------------------------+
+| :math:`E(\rho, T)`       | Energy               | Density, Temperature   |
++--------------------------+----------------------+                        |
+| :math:`P(\rho, T)`       | Pressure             |                        |
++--------------------------+----------------------+------------------------+
+| :math:`\rho(P, T)`       | Density              | Pressure, Temperature  |
++--------------------------+----------------------+                        |
+| :math:`E(P, T)`          | Energy               |                        |
++--------------------------+----------------------+------------------------+
+| :math:`C_V(\rho, T)`     | Constant Volume      | Density, Temperature   |
++--------------------------+ Specific Heat        +------------------------+
+| :math:`C_V(\rho, E)`     | Capacity             | Density, Energy        |
++--------------------------+----------------------+------------------------+
+| :math:`B_S(\rho, T)`     | Isentropic Bulk      | Density, Temperature   |
++--------------------------+ Modulus              +------------------------+
+| :math:`B_S(\rho, E)`     |                      | Density, Energy        |
++--------------------------+----------------------+------------------------+
+| :math:`\Gamma(\rho, T)`  | Gruneisen Parameter  | Density, Temperature   |
```

Comment:
Hmm using `\Gamma` as the Gruneisen parameter I think is going to be confusing for hydro folks.  Maybe `\Gamma_1` (or whatever subscript it actually is?) might be better. Or alternatively, just use `(\Gamma - 1)` everywhere?

Reply:
You're going to need to define some quantities there because as an EOS person I don't understand `\Gamma_1` and if your `\Gamma` is the adiabatic exponent, I'm pretty sure that the Gruneisen parameter is only equal to `\Gamma - 1` for an ideal gas. While I have seen both lowercase `\gamma` as well as uppercase `\Gamma` for the Gruneisen parameter, I think outside of astrophysics the adiabatic exponent is almost exclusively represented by small `\gamma` so `\Gamma` seems logical for the Gruneisen parameter.

Find me some resources defining the parameters you mention and I'll consider it. But I think changing it will make it confusing for the EOS folks.

Reply:
I can't find the reference for \Gamma_1 at the moment.

How about a compromise: use lower case (\gamma-1) in the ideal gas formula and say that in the special case of an ideal gas, this is equal to the Gruneisen parameter.

Also note that the Gruneisen parameter is referred to as gm1 throughout the implementation of singularity-eos for this reason.

Reply:
I'll note that `\Gamma = \gamma - 1` for an ideal gas.

Yes I've seen that in the code, but I think the `\Gamma_1` notation is specifically for astrophysics. The only reference I've found has been here:
https://en.wikipedia.org/wiki/Heat_capacity_ratio#Adiabatic_process.

Aside from E being internal energy, I also think it's good to follow the nomenclature of Menikoff and Plohr since I would consider it a seminal paper at the intersection of EOS and hydrodynamics. They use `\gamma` for the adiabatic exponent and `\Gamma` for the Gruneisen parameter as well.

I think it would be very helpful to add a disambiguation section where we describe the different Gammas and how they are related. If you can find me something that documents your understanding, I'll put it in that section. I'll start with a note about `\gamma` vs `\Gamma` there as well.

Reply:
Menikoff and Plohr for reference:
https://journals.aps.org/rmp/abstract/10.1103/RevModPhys.61.75

Reply:
I think disambiguating across all communities is kind of a lost cause. I'm happy enough with the notation with the big and little gammas. I think with the table that's clear enough.

I don't think tying ourselves to the notation of a single paper is wise, but you can mention menikhoff and plohr and your primary source notation wise if you like.

Maybe the better disambiguation we should add is a translation between code and math notation. I.e., the Gruneisen thing. And that energy is often written as sie in the code. Things like that.

Reply:
Well, I wouldn't mind defining multiple Gammas if they're commonly used as you said. My first encounter with them was talking with you and Josh so I'd appreciate knowing what other notations are out there.

Reply:
As for relating code variables to thermodynamic quantities, I think the API section is a better place for that.

Reply:
@Yurlungur is there anything else you'd like me to do to address this issue?